src/main/java/org/fundacion/jala/converter/controller/config/SwaggerConfig.java

```diff
@@ -63,4 +100,22 @@ private ApiInfo apiDetails() {
                 "localhost:8080/api/license",
                 Collections.emptyList());
     }
+
+    /**
+     * Creates an API key.
+     *
+     * @return the API key.
+     */
+    private ApiKey apiKey() {
+        return new ApiKey("JWT", "Authorization", "header");
+    }
+
+    /**
+     * hello
+     * @return
```

Comment:
It should be have a blank line between desciption and @return.

Reply:
corrected it.